Pipfile

```diff
@@ -6,15 +6,17 @@ name = "pypi"
 [dev-packages]
 pytest = "==4.3.0"
 pytest-django = "==3.4.8"
-
-[packages]
-wagtail = ">=2.3"
+black = "==18.9b0"
+pre-commit = ">=1.14.4,<1.15"
 bumpversion = "==0.5.3"
 codecov = ">=2.0.0"
 coverage = "==4.3.4"
 "flake8" = ">=2.1.0"
 mock = ">=1.0.1"
 tox = ">=1.8"
+
+[packages]
+wagtail = ">=2.3"
```

Comment:
do we want to limit this to `<2.4` as well?

Reply:
`Omni-Blocks` had already undergone a wagtail upgrade so this wasn't something that I had changed, but yes I'll add `<=2.4`

Reply:
Fixed with cfb4207dec3db1f158c1acc9cee812bfa667093a